src/dev.sh

```diff
@@ -28,6 +28,12 @@ RUNNER_VERSION=$(cat runnerversion)
 
 pushd "$SCRIPT_DIR"
 
+if which pwsh; then
+    POWERSHELL_VERSION="pwsh"
```

Comment:
```suggestion
    POWERSHELL="pwsh"
```

what about just calling it `POWERSHELL`?


`POWERSHELL_VERSION` sounds like a version `10.1.1`, etc.